src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/ConfigurationRepository.java

```diff
@@ -147,7 +158,7 @@ public void run() {
                                         ConfigHelper.uploadFile(client, cd+"nodes_dn.yml", opendistrosecurityIndex, CType.NODESDN, DEFAULT_CONFIG_VERSION, populateEmptyIfFileMissing);
                                         LOGGER.info("Default config applied");
                                     } else {
-                                        LOGGER.error("Can not create {} index", opendistrosecurityIndex);
+                                        LOGGER.error("Can not use or create {} index", opendistrosecurityIndex);
```

Comment:
Is there a way to get an error message with the details of why the index can't be created? There is no yet attempt to "use" the index, is the error message should be "Failed to create {} index"?

Reply:
Well I just wanted to mean that "not ok" means, it could not attempt to use the index to update the documents. 
I don't mind changing the grammar. Let me know

Reply:
Do you mean to say "Request to create index {} is not acknowledged. The index may not be usable to load dynamic configuration."?